cassandra/connection.py

```diff
@@ -44,6 +44,7 @@
                                 RegisterMessage, ReviseRequestMessage)
 from cassandra.util import OrderedDict
 
+MIN_LONG = -(2 ** 63)
```

Comment:
Since this is Python code, consider naming it something else maybe.

Python used to have `long` type and `sys.maxint` back in Python2 days; The value of the latter roughly corresponded to target architecture pointer size...

To avoid confusion for Python-folk, consider naming this constant something scylla-specific, like SCYLLA_XX_BIAS, or just inline it in the single place where it's used.

Reply:
it's actually copied from a different part of this source code, including the name.
and I didn't imported from there, cause of cyclic dependencies.

looking at it again, I think it would be better to import from `murmur3.INT64_MIN`

Reply:
done